### PR TITLE
Add price identifiers for AUDUSD,BRLUSD,MXNUSD,COPUSD,CZKUSD,SEKUSD,P…

### PR DESCRIPTION
**Pull request by SayWhatDev**
…LNUSD and TRYUSD

### CODE REVIEW

**Review by therealhexi — DISMISSED** [review]
Looks good to me

File: UMIPs/umip_add_new_price_identifiers_jarvis.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+Documentation for the API can be found [here](https://fcsapi.com/document/forex-api).
+
+- 500 API Calls with Free Plan
+- Live price data accessible with Free Plan - Updated every 60 seconds or less
+- Historical data accessible with the Basic Paid Plan - 10$ per month
+
+Example requests:
+
+- Live Rate for AUD/USD: `https://fcsapi.com/api-v3/forex/latest?symbol=AUD/USD&access_key=api_key`
+- Historical price for a certain date for SGD/USD - gives back the OHLC: `https://fcsapi.com/api-v3/forex/history?symbol=AUD/USD&period=1D&from=2020-12-23&to=2020-12-24&access_key=api_key`
```
Comment (Reinis-FRP) [inline UMIPs/umip_add_new_price_identifiers_jarvis.md:151]:
```suggestion
- Historical price for a certain date for AUD/USD - gives back the OHLC: `https://fcsapi.com/api-v3/forex/history?symbol=AUD/USD&period=1D&from=2020-12-23&to=2020-12-24&access_key=api_key`
```

File: UMIPs/umip_add_new_price_identifiers_jarvis.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+Apart from the weekend, there is little to no difference in prices on liquid major Forex pairs like AUDUSD, so any price feed could be used; however, for convenience, we recommend using the one of TraderMade.
+
+## Implementation
+
+Historical  AUDUSD, BRLUSD, MXNUSD, COPUSD, CZKUSD, SEKUSD, PLNUSD and TRYUSD prices from TraderMade are available in minute increments. Price requests should use the minute price that is nearest and later than the price request timestamp. To do this, voters should use the open price of the OHLC period that the price request timestamp falls in. TraderMade endpoints are queried based on the OHLC period's close time.
+
+As an example, a request for an AUDUSD price at 2020-11-11-01:52:16 should query for the period ending at 2020-11-11-01:53:00 and use the open price.
+
+The TraderMade endpoint used would be: https://marketdata.tradermade.com/api/v1/minute_historical?currency=AUDUSD&date_time=2020-11-11-13:01&api_key=apikey
```
[the remark this answers is not in the document]
Comment (SayWhatDev) [reply UMIPs/umip_add_new_price_identifiers_jarvis.md:88]: @therealhexi Hey yes, they don't provide seconds, so if the minute historical price is used then the timestamp in that example should be `01:53`

[the remark this answers is not in the document]
Comment (SayWhatDev) [reply UMIPs/umip_add_new_price_identifiers_jarvis.md:88]: Will fix it and push a commit :)

**Review by Reinis-FRP — APPROVED** [review]

**Pull request merged**